src/main/fc/controlrate_profile.c

```diff
@@ -39,7 +39,7 @@ PG_REGISTER_ARRAY_WITH_RESET_FN(controlRateConfig_t, CONTROL_RATE_PROFILE_COUNT,
 void pgResetFn_controlRateProfiles(controlRateConfig_t *controlRateConfig)
 {
     for (int i = 0; i < CONTROL_RATE_PROFILE_COUNT; i++) {
-        RESET_CONFIG(const controlRateConfig_t, &controlRateConfig[i],
```

Comment:
What's the reason for removing the `const` from these?

Reply:
why this change is needed are descriped in the pull request message.